packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## NEXT
+## 0.3.2+2
 
+* Migrates to `dart:ui_web` APIs.
+* Updates minimum supported SDK version to Flutter 3.13.0/Dart 2.19.
```

Comment:
Delete below line?
Might as well make min Dart 3.0 – right? Then you can use new language features!

Reply:
I'll let @stuartmorgan chime in here since I'm not sure what the guidelines are here..

Reply:
That's why we have CI enforcement instead of guidelines :)
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8771408187227288369/+/u/Run_package_tests/pubspec_validation/stdout

Reply:
Aha! I'll fix it.

Reply:
Done